tests/listobjects/listobjects_test.go

```diff
@@ -24,10 +24,6 @@ func TestCheckMySQL(t *testing.T) {
 }
 
 func testRunAll(t *testing.T, engine string) {
-	testListObjects(t, engine)
-}
-
-func testListObjects(t *testing.T, engine string) {
 	cfg := run.MustDefaultConfigWithRandomPorts()
 	cfg.Log.Level = "none"
 	cfg.Datastore.Engine = engine
```

Comment:
One of the server configs is `ListObjectsMaxResults`. In an upcoming PR, I will add testing for this parameter, so I will need each test to create a new server. Can you make the change in this PR so I don't have a lot of merge conflicts?